docs/src/main/asciidoc/sending-emails.adoc

```diff
@@ -276,11 +276,6 @@ quarkus.mailer.username=YOUREMAIL@gmail.com
 quarkus.mailer.password=YOURGENERATEDAPPLICATIONPASSWORD
 ----
 
-== Using SSL with native executables
-
-Note that if you enable SSL for the mailer and you want to build a native executable, you will need to enable the SSL support.
-Please refer to the native-and-ssl-guide.html[Using SSL With Native Executables] guide for more information.
-
```

Comment:
Could you also update the list of extensions enabling SSL automatically here: https://quarkus.io/guides/native-and-ssl-guide ?

It's just after the sentence `As SSL is ipso facto the standard nowadays, we decided to enable its support automatically for some of our extensions:`.

I think we miss some of the recently added (Neo4j comes to mind). We should s/SmallRye REST client/REST client/ too. And maybe we should add the name of the extensions in parentheses (`quarkus-mailer`).

Reply:
Good catch. I’ll update that part.